library/CMakeLists.txt

```diff
@@ -47,7 +47,20 @@ if (ziti_DEVELOPER_MODE)
     endif ()
 endif ()
 
-find_package(json-c CONFIG REQUIRED)
+find_package(json-c CONFIG QUIET)
+if(NOT json-c_FOUND)
+    find_package(PkgConfig REQUIRED)
```

Comment:
- PkgConfig is already loaded here
- pkg_check_modules already creates a target (see how it loads `libsodium`). make it similar to that